.github/workflows/commitlint.yml

```diff
@@ -1,13 +1,13 @@
 name: "Commit Linter"
 on:
-  pull_request:
+  # Only pull_request and push honor [skip ci].  Since this workflow must pass
+  # to merge a PR, it can't be skipped, so use pull_request_target
+  pull_request_target:
     types: [opened, synchronize, reopened, ready_for_review]
 jobs:
   lint-commits:
     runs-on: ubuntu-latest
```

Comment:
Minor nitpick, but we might need to change permissions. When I tested it all permissions were set to `write`, even though I used a different github account for the PR. I'm a bit unsure about this since it might depend on repository setting, but just as a heads up.